ovos_persona_server/ollama.py

```
@@ -0,0 +1,373 @@
+                        "created_at": ts,
+                        "message": {"role": "assistant", "content": chunk},
+                        "done": False,
+                        # Update total_duration dynamically for streaming chunks
+                        "total_duration": int((time.time() - start_time) * 1_000_000_000),
+                        "load_duration": streaming_load_duration,
+                        "prompt_eval_count": streaming_prompt_eval_count,
+                        "prompt_eval_duration": streaming_prompt_eval_duration,
+                        "eval_count": streaming_eval_count,
+                        "eval_duration": streaming_eval_duration  # This would be cumulative in a real scenario
+                    }) + "\n"
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)